test/test_transforms.py

```diff
@@ -1997,5 +1832,175 @@ def test_random_order():
+def test_random_erasing():
+    img = torch.ones(3, 128, 128)
+
+    t = transforms.RandomErasing(scale=(0.1, 0.1), ratio=(1 / 3, 3.))
+    y, x, h, w, v = t.get_params(img, t.scale, t.ratio, [t.value, ])
+    aspect_ratio = h / w
+    # Add some tolerance due to the rounding and int conversion used in the transform
+    tol = 0.05
+    assert (1 / 3 - tol <= aspect_ratio <= 3 + tol)
+
+    aspect_ratios = []
+    random.seed(42)
+    trial = 1000
+    for _ in range(trial):
+        y, x, h, w, v = t.get_params(img, t.scale, t.ratio, [t.value, ])
+        aspect_ratios.append(h / w)
+
+    count_bigger_then_ones = len([1 for aspect_ratio in aspect_ratios if aspect_ratio > 1])
+    p_value = stats.binom_test(count_bigger_then_ones, trial, p=0.5)
+    assert p_value > 0.0001
+
+    # Checking if RandomErasing can be printed as string
+    t.__repr__()
+
+
+def test_random_rotation():
+
+    with pytest.raises(ValueError):
+        transforms.RandomRotation(-0.7)
+        transforms.RandomRotation([-0.7])
+        transforms.RandomRotation([-0.7, 0, 0.7])
+
+    # assert fill being either a Sequence or a Number
+    with pytest.raises(TypeError):
+        transforms.RandomRotation(0, fill={})
+
+    t = transforms.RandomRotation(0, fill=None)
+    assert t.fill == 0
+
+    t = transforms.RandomRotation(10)
+    angle = t.get_params(t.degrees)
+    assert (angle > -10 and angle < 10)
+
+    t = transforms.RandomRotation((-10, 10))
+    angle = t.get_params(t.degrees)
+    assert (-10 < angle < 10)
+
+    # Checking if RandomRotation can be printed as string
+    t.__repr__()
+
+    # assert deprecation warning and non-BC
+    with pytest.warns(UserWarning, match=r"Argument resample is deprecated and will be removed"):
+        t = transforms.RandomRotation((-10, 10), resample=2)
+        assert t.interpolation == transforms.InterpolationMode.BILINEAR
+
+    # assert changed type warning
+    with pytest.warns(UserWarning, match=r"Argument interpolation should be of type InterpolationMode"):
+        t = transforms.RandomRotation((-10, 10), interpolation=2)
+        assert t.interpolation == transforms.InterpolationMode.BILINEAR
+
+
+def test_randomperspective():
+    for _ in range(10):
+        height = random.randint(24, 32) * 2
+        width = random.randint(24, 32) * 2
+        img = torch.ones(3, height, width)
+        to_pil_image = transforms.ToPILImage()
+        img = to_pil_image(img)
+        perp = transforms.RandomPerspective()
+        startpoints, endpoints = perp.get_params(width, height, 0.5)
+        tr_img = F.perspective(img, startpoints, endpoints)
+        tr_img2 = F.to_tensor(F.perspective(tr_img, endpoints, startpoints))
+        tr_img = F.to_tensor(tr_img)
+        assert img.size[0] == width
+        assert img.size[1] == height
+        assert (torch.nn.functional.mse_loss(tr_img, F.to_tensor(img)) + 0.3 >
+                torch.nn.functional.mse_loss(tr_img2, F.to_tensor(img)))
+
+
+def test_randomperspective_fill():
```

Comment:
I wanted to parametrize this with pytest. But that won't be possible with the current test implementation. See my comment below.